Source/Spray/BreakupSplash/WallFilm.H

```diff
@@ -158,20 +158,20 @@ calculateFilmSource(
     film_height = new_mass / (rho_film * film_area);
     if (film_height > min_height) {
       if (SPRAY_FUEL_NUM > 1) {
-        rho_film = 0.;
+        // rho_film = 0.;
         for (int spf = 0; spf < SPRAY_FUEL_NUM; ++spf) {
           Y_film[spf] = amrex::min(
             1., amrex::max(
                   (Y_film[spf] * film_mass + mi_dot[spf] * dt) / new_mass, 0.));
           if (Y_film[spf] < 1.E-12) {
             Y_film[spf] = 0.;
           }
-          rho_film += Y_film[spf] / fdat.rhoL(T_film, spf);
+          // rho_film += Y_film[spf] / fdat.rhoL(T_film, spf);
         }
-        rho_film = 1. / rho_film;
-      } else {
-        rho_film = fdat.rhoL(T_film, 0);
-      }
+        // rho_film = 1. / rho_film;
+      } // else {
+        //  rho_film = fdat.rhoL(T_film, 0);
+      //}
```

Comment:
@ldowen do you know why these are computed but not used? Is that a bug or just something that changed?

Reply:
These were recomputed in case this whole update was in a sub-cycle. That clearly isn't the case so they are unnecessary.